quake1_example/scene/player.tscn

```diff
@@ -14,23 +14,21 @@ collision_layer = 3
 floor_constant_speed = true
 floor_snap_length = 0.5
 platform_on_leave = 2
-script = ExtResource("1_rapov")
+script = ExtResource("1_qw014")
 
 [node name="col" type="CollisionShape3D" parent="."]
 shape = SubResource("BoxShape3D_c8hfp")
 
 [node name="around" type="Node3D" parent="."]
-transform = Transform3D(1, 0, 0, 0, 1, 0, 0, 0, 1, 0, 0.688, 0)
+transform = Transform3D(1, 0, 0, 0, 1, 0, 0, 0, 1, 2.08165e-12, 0.688, 2.08165e-12)
```

Comment:
`2.08165e-12` shows up as `0` in editor, maybe this is a godot bug, or floating point rounding? 
<img width="273" alt="Screenshot 2023-06-28 at 4 45 06 PM" src="https://github.com/gongpha/gdQmapbsp/assets/315297/e655b2f8-09af-47b9-a639-4fdcac74a267">

Reply:
Likely the latter. imo, that number seems very precise enough to me.

Reply:
I tried updating by typing `0` in the editor but it doesn't want to actually set it to `0` just keeps the odd number...

Reply:
Confirmed that this is an unusual behavior. But occurred on the Y field instead
![image](https://github.com/gongpha/gdQmapbsp/assets/13400398/c5caa217-fe0a-4514-9d9e-af72b85194ab)